scripts/SANS/sans/test_helper/mock_objects.py

```diff
@@ -26,6 +27,25 @@ def create_mock_masking_table():
     return view
 
 
+def create_mock_beam_centre_tab(model = None):
```

Comment:
you should mock the view instead of setting it equal to the model. See https://docs.python.org/3/library/unittest.mock.html, you can check the values the functions are called with too